kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -138,17 +138,48 @@ domain.weblogic.oracle/domain1 patched
 
 The `rollCluster.sh` script updates the value of `spec.clusters[<cluster-name>].restartVersion` attribute of the domain resource. The operator will then do a rolling restart of the WebLogic cluster Server Pods once the value of `spec.clusters[<cluster-name>].restartVersion` is updated. You can provide the new value of `restartVersion` as a parameter to the script or the script will automatically generate a new value to trigger the rolling restart. See the script `usage` information by using the `-h` option.
 
-```shell
+```
 $ rollCluster.sh -c cluster-1 -d domain1 -n weblogic-domain-1
 [2021-03-24T04:03:27.521 UTC][INFO] Patching restartVersion for cluster 'cluster-1' to '2'.
 domain.weblogic.oracle/domain1 patched
 [2021-03-24T04:03:27.669 UTC][INFO] Successfully patched replicas for cluster 'cluster-1'!
 ```
 
 Use the following command to roll the WebLogic Cluster Servers with a specific `restartVersion`:
-```shell
+```
 $ rollCluster.sh -r v2 -c cluster-1 -d domain1 -n weblogic-domain-1
 [2021-03-24T13:46:16.833 UTC][INFO] Patching restartVersion for cluster 'cluster-1' to 'v2'.
 domain.weblogic.oracle/domain1 patched
 [2021-03-24T13:46:16.975 UTC][INFO] Successfully patched replicas for cluster 'cluster-1'!
 ```
+
+### Scripts to restart a WebLogic Server in a domain
+The `restartServer.sh` can be used to restart a WebLogic Server in a domain. This script restarts the server by deleting the Server Pod for the WebLogic Server instance.
+```
+$ restartServer.sh -s managed-server1 -d domain1 -n weblogic-domain-1
+[2021-03-24T22:20:22.498 UTC][INFO] Initiating restart of 'managed-server1' by deleting server pod 'domain1-managed-server1'.
+[2021-03-24T22:20:37.614 UTC][INFO] Server restart succeeded !
+```
+
+### Scripts to explicitly initiate introspection of a WebLogic domain
+
+The `introspectDomain.sh` script can be used to explicitly initiate the introspection of a WebLogic domain managed by the operator.  This script updates the value of `spec.introspectVersion` attribute of the domain resource to explicitly initiate the introspection. You can provide the new value for `spec.introspectVersion` as a parameter to the script or the script will automatically generate a new value to trigger the introspection. See the script `usage` information by using the `-h` option.
+
+```
+$ introspectDomain.sh -d domain1 -n weblogic-domain-1
+[2021-03-24T21:37:55.989 UTC][INFO] Patching introspectVersion for domain 'domain1' to '1'.
+domain.weblogic.oracle/domain1 patched
+[2021-03-24T21:37:56.110 UTC][INFO] Successfully patched introspectVersion for domain 'domain1'!
+```
+
+Use the following command to roll the Server Pods in a WebLogic domain with a specific `introspectVersion`:
```

Comment:
_This  text is misleading because the pods may or may not roll after a re-introspect.  The life-cycle doc for introspectVersion discusses the details.  So maybe reword this to something like:_

Use the following command to rerun a domain's introspect job. The resulting behavior depends on your domain home source type and other factors, see [link to the life-cycle doc for introspect version] for details.